packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -203,5 +204,6 @@ export const menuVariables = (siteVars: any): MenuVariables => {
     indicatorColorHover: siteVars.colors.grey[500],
 
     subMenuBorderColor: siteVars.colorScheme.default.borderTransparent,
+    subMenuIconColor: siteVars.colors.brand[600],
```

Comment:
I think you'll want to use a token instead of `brand[600]` so that it will automatically swap colors in the different themes

Reply:
It *might* be `Brand Background Hover`, but I would verify with Daisy

Reply:
Is this for the icon? 

`Brand Foreground Hover `should work :) (I'm assuming the background is switching to cyan for high contrast?)
![image](https://user-images.githubusercontent.com/48726002/193649974-7b377051-bee5-40e2-8496-521c626d15b7.png)

Reply:
Yes It is for the icon color, when hovering in the item

Reply:
Here's the link of the PR: https://fluentuipr.z22.web.core.windows.net/pull/25012/react-northstar/components/menu/definition